cdap-ui/app/tracker/main.js

```diff
@@ -250,6 +250,35 @@ angular
     var activeThemeClass = caskTheme.getClassName();
     var dataSource = new MyCDAPDataSource($scope);
     getVersion();
+    this.eventEmitter = window.CaskCommon.ee(window.CaskCommon.ee);
+    this.pageLevelError = null;
+    const {globalEvents} = window.CaskCommon;
+
+    const handlePageLevelError = (error) => {
+      // This function parses receiveing error messages and converts it to a
+      // format that page level error supports
+      let message = null;
```

Comment:
Seems like duplicate code. Can we export it as a common utility function and use it here?

Reply:
Fixed.